doc/src/explanation/active-deprecations.md

```diff
@@ -435,6 +435,9 @@ than an element method.
 Symbolic functions in ``ntheory`` have been moved to ``functions``.
 Specifically, the following functions.
 
+* ``sympy.ntheory.factor_.primenu``
+* ``sympy.ntheory.factor_.primeomega``
+* ``sympy.ntheory.factor_.totient``
 * ``sympy.ntheory.residue_ntheory.mobius``
```

Comment:
The deprecation note here should be a bit clearer about what is happening from a user perspective.

All of these functions can be imported like `from sympy import mobius` which continues to work just fine after these changes as well as it did before. What is relevant for users/downstream is that code that does `from sympy.ntheory.residue_ntheory import mobius` should be changed to `from sympy import mobius`.

Reply:
I added a description.